client/src/components/VersionedBadge/VersionedBadge.scss

```diff
@@ -0,0 +1,13 @@
+.versioned-badge {
+  padding-left: 0;
+  padding-right: 0;
+
+  &--draft,
+  &--modified {
+    color: $orange;
```

Comment:
@sachajudd is this the bit you were talking about? We have `$state-draft` and `$state-modified`, do we need to add `$state-live` and `$state-archived` as well (with their respective background equivalents too)?

Reply:
Yeah sounds good!

Reply:
@sachajudd Are we happy with the live and archived colour in the screen shot? 
* Live uses #008a00
* Archived default to #43536d which is the default for the Badge component

I'll create them matching `$state-live` and `$state-archived` variables in `_variables.scss`

Reply:
Yeah so by the looks of it we can use these variables for the same colours.
$state-archived: $gray-700;
$state-live: $green;

Reply:
Done.

Reply:
@maxime-rainville have you pushed up your changes I can't see the variable $state-live being added or used in VersionedBadge.scss

Reply:
There’s also a badge.scss file in admin/client/components/badge that has status-archived styles which could be replaced to use the new variable you've created ($status-archived).